dashboard/cron.py

```diff
@@ -196,6 +272,11 @@ def util_function(sql_string, mysql_table):
         dbname=UDW_DATABASE)
 
     df = pd.read_sql(sql_string, udw_conn)
+    # Sql returns boolean value so grouping course info along with it so that this could be stored in the DB table.
```

Comment:
not quite sure about the comment line "Sql returns boolean value".

Reply:
When the `weight_consideration` function run it will return a boolean value (i.e if course consider weights for grade calculation or not). The 3 lines below after that comment is adding course name associated with the boolean return. That's what the comment mean. I couldn't figure out an elegant sql solution to combine boolean value with course_id so I took the help of `dataframes` in doing that and adding it to the stu_dash tables